### ISSUE TITLE
Error connecting to Redis server at 127.0.0.1:35709

### ISSUE THREAD

**Issue by Tabbi7** [ray-project/ray#8389]
### What is the problem?

I'm trying to use the framework Flow included in flow-project (https://github.com/flow-project/flow) that has a problem with Ray. In fact, I have the error "Waiting for redis server at 127.0.0.1:35709 to respond...": I'm not able to connect to the Redis server.

<img width="1163" alt="Schermata 2020-05-10 alle 13 23 18" src="https://user-images.githubusercontent.com/17814193/81497967-5f653380-92c2-11ea-8b58-d0ed01274743.png">

**Comment by rkooo567** [comment]
Is it like an example code in the flow-project repository? Can you give us a link to that jupyter notebook?

**Comment by Tabbi7** [comment]
Yes, it is a problem related to the flow-project. How can I link that Jupiter notebook? I'm attaching a zip containing the tutorial 3 my Flow environment. I hope you can replicate the problem in your localhost. Thank you so much!

[Notebook.zip](https://github.com/ray-project/ray/files/4608029/Notebook.zip)

**Comment by rkooo567** [comment]
Can you create a gist instead?

**Comment by Tabbi7** [comment]
Here you have the gist: https://gist.github.com/Tabbi7/be0e3adf334cf6b8837418b0c8305c43

**Comment by cszhu** [comment]
Hello! This P3 issue has seen no activity in at least a year. It will be closed in 2 weeks as part of ongoing cleanup efforts.

Please remove the `pending-cleanup` label if you believe this issue should remain open.

Thanks for contributing to Ray!
